tests/summ_ai_api/summ_ai_test.py

```diff
@@ -231,9 +231,11 @@ async def test_translate_text_field_internal_server_error(
     await my_api_client.translate_text_field(session, text_field)
 
     errors = tuple(
-        map(
-            lambda record: record.message,
-            filter(lambda record: record.levelname == "ERROR", caplog.records),
+        (
+            record.message
+            for record in filter(
+                lambda record: record.levelname == "ERROR", caplog.records
+            )
```

Comment:
```suggestion
        (
            record.message
            for record in caplog.records
            if record.levelname == "ERROR"
            )
```

I think we could also get rid of the `filter` call

Reply:
These had one closing paren too many but I fixed and squashed.